vendor/github.com/google/cadvisor/info/v1/machine.go

```diff
@@ -263,6 +263,7 @@ func (m *MachineInfo) Clone() *MachineInfo {
 		NumSockets:       m.NumSockets,
 		CpuFrequency:     m.CpuFrequency,
 		MemoryCapacity:   m.MemoryCapacity,
+		SwapCapacity: 	  m.SwapCapacity,
```

Comment:
For vendor updates, you should update it in `github.com/google/cadvisor` and then update the dependency lib like https://github.com/kubernetes/kubernetes/pull/118774.